api/src/main/java/com/nicknathanjustin/streamercontracts/contracts/ContractsApiController.java

```diff
@@ -61,4 +67,75 @@ public ResponseEntity voteOnContract(@RequestBody @NonNull final ContractVoteReq
 
         return new ResponseEntity(HttpStatus.OK);
     }
+
+    @RequestMapping(path = "listStreamerBounties/{page}/{pageSize}", method = RequestMethod.GET)
+    public ResponseEntity listContractsForStreamer(
+            @PathVariable final int page,
+            @PathVariable final int pageSize,
+            @RequestParam("state") final Optional<String> optionalState,
+            @RequestParam("username") final Optional<String> optionalUsername) {
```

Comment:
I think @Nullable is a little better than an Optional for this case. Similar reasoning to the conversation we had on the previous CR for why optionals as inputs to serviceMethods dont really add much. The same logic applies here.

@Nullable only has 2 states to check for Null, or not null

Optional has 3 states. The optional could be null. It could hold null. or the valur could be there.